easybuild/easyblocks/t/torchvision.py

```diff
@@ -78,6 +78,11 @@ def configure_step(self):
             if cuda_cc:
                 env.setvar('TORCH_CUDA_ARCH_LIST', ';'.join(cuda_cc))
 
+        libjpeg = get_software_root('libjpeg-turbo')
+        if libjpeg and 'TORCHVISION_INCLUDE' not in self.cfg['preinstallopts']:
+            vision_include = 'TORCHVISION_INCLUDE="$EBROOTLIBJPEGMINTURBO/include:$TORCHVISION_INCLUDE"'
```

Comment:
Won't `$TORCHVISION_INCLUDE` always be empty in the build environment?

Also, shouldn't this be `export TORCHVISION_INCLUDE=... && `?

Reply:
(i accidentally edited your post, should have been a quote reply)

Reply:
@boegel i replaced it with an envvar as you seem to prefer that, and also added a sanity check in https://github.com/easybuilders/easybuild-easyblocks/pull/3322/commits/671331c1c787f041c6284d8bd9dcb33f6f54d03b